function-support.html

```diff
@@ -947,6 +931,21 @@ <h2 id="annotation">Annotation</h2>
 		</tr>
 	</table>
 	
+	<div class="indented">
+		<p><code>\tag{hi} x+y^{2x}</code></p>
+		<p>\[\tag{hi} x+y^{2x} \]</p>
+		<p><code>\tag*{hi} x+y^{2x}</code></p>
+		<p>\[\tag*{hi} x+y^{2x} \]</p>
+		<p><code>\tag</code> is not yet supported inside the <code>{gather}</code> environment.</p>

```

Comment:
`{gather}` is a top-level environment, which KaTeX doesn't support.  So maybe we should say `{gathered}`.  And should probably also list `{aligned}` and `{alignedat}`; I believe `amsmath` supports `\tag` in `equation`, `gather`, `multiline`, `split`, `align`, `alignat`, `ftalign`, but among those, we only have `gathered`, `aligned`, `alignedat`.